tests/cli/test_suite.py

```diff
@@ -4476,3 +4480,34 @@ def test__process_suite_edit_flags_and_prompt(
                 }
             ),
         ]
+
+
+def test_suite_new_load_jupyter_configured_asset_sql_data_connector_missing_data_asset(
+    sqlite_missing_data_asset_data_context,
+):
+    context: DataContext = sqlite_missing_data_asset_data_context
+
+    interactive_mode, profile = _process_suite_new_flags_and_prompt(
+        context=context,
+        usage_event_end=None,
+        interactive_flag=True,
+        manual_flag=False,
+        profile=False,
+        batch_request=None,
+    )
+
+    try:
+        _suite_new_workflow(
```

Comment:
This was erroring out due to missing `data_asset_name` previously

Reply:
aha yes I ran into this quite a bit, thanks for patching it!